src/org/sparkboard/slack/screens.clj

```diff
@@ -0,0 +1,294 @@
+(ns org.sparkboard.slack.screens
+  (:require [clojure.pprint :as pp]
+            [org.sparkboard.firebase.jvm :as fire-jvm]
+            [org.sparkboard.js-convert :refer [clj->json json->clj]]
+            [org.sparkboard.server.env :as env]
+            [org.sparkboard.slack.api :as slack]
+            [org.sparkboard.slack.view-examples :as examples]
+            [org.sparkboard.slack.slack-db :as slack-db]
+            [org.sparkboard.slack.urls :as urls]
+            [org.sparkboard.slack.view :as v]
+            [org.sparkboard.util.future :refer [try-future]]
+            [taoensso.timbre :as log]))
+
+(def team-messages
+  {:welcome
+   {:label "Welcome message"
+    :default "*Welcome here!* Please connect your Sparkboard account to continue:"}
+   :welcome-confirmation
+   {:label "Confirmation of account linking"
+    :default (str "*Thanks!* You're ready to go. Projects on Sparkboard "
+                  "automatically get a linked channel created here on Slack.")}})
+
+(defn team-message [context k]
+  (or (get-in context [:slack/team :custom-messages k])
+      (get-in team-messages [k :default])))
+
+(v/defview link-account [context]
```

Comment:
Some of the explanation text in this PR might make sense here as a comment, or docstring on `defview`. Possibly in a future pass.

Reply:
Good point. I'll add some more comments & docstrings